client/lib/wp/sync-handler/test/cache-index.js

```diff
@@ -30,18 +30,13 @@ describe( 'cache-index', () => {
 	describe( '#getAll', () => {
 		beforeEach( clearLocal );
 
-		it( 'should return undefined for empty localforage', ( done ) => {
+		it( 'should return undefined for empty localforage', () => {
 			return cacheIndex.getAll()
 				.then( res => {
-					try {
```

Comment:
I was curious about the changes and dug up the changelog. 
https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#300--2016-07-31
looks good